mhs/common/mhs_common/state/tests/test_work_description.py

```diff
@@ -98,7 +100,7 @@ def test_constructor(self):
             wd.DATA: {
                 wd.VERSION_KEY: 1,
                 wd.LATEST_TIMESTAMP: '11:00',
-                wd.STATUS: wd.MessageStatus.OUTBOUND_MESSAGE_PREPARED
+                wd.INBOUND_STATUS: wd.MessageStatus.OUTBOUND_MESSAGE_PREPARED,
```

Comment:
I'm not sure it affects this test case, but does it make sense for the inbound status to be `OUTBOUND_MESSAGE_PREPARED`?